.env

```diff
@@ -0,0 +1 @@
+DATABASE_URL="Server=localhost;port=5432;Database=WhaleSpottingDb;user id=postgres;password=techswitch;"
```

Comment:
We don't want to commit this :) it should be covered by the gitignore! If it isn't it's worth checking the gitignore is correct